db/migrations/20230713142344_endpoint_name.sql

```diff
@@ -0,0 +1,9 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE unweave.endpoint ADD COLUMN name text UNIQUE;
```

Comment:
I think we want the name to be unique per project and no globally unique. The url can have a the username as a prefix to enable globally unique URLS

Reply:
In sites like Heroku and Netlify, the names are globally unique so they can provide `<name>.appdomain.com`

It would provide cleaner URLs than if we had to always prefix with the username.

I’m guessing Vercel and Railsway, but haven’t checked.